### DISCUSSION TITLE
Assign (team) reviewers to PRs

### DISCUSSION THREAD

**Discussion by tomahawk28** [renovatebot/renovate#11217] [Request Help]
Hi, I'm trying to automate PR creation with our team name as `team-batman-engineer`, so I updated our renovate.json as below. 
```
  "reviewers": [ "team:giantswarm/team-batman-engineers" ],
```
https://github.com/giantswarm/cluster-apps-operator/blob/30f045941c436e7eda54d4513f7cac442dad7378/renovate.json#L28

I also see it worked from logs in renovate dashboard. 

```
DEBUG: Added reviewers(branch="renovate/etc-modules")
{
  "reviewers": [
    "team:giantswarm/team-batman-engineers"
  ]
}
DEBUG: Created Pull Request #59(branch="renovate/etc-modules")
```

However, it always ends up with an empty reviewer. see this PR https://github.com/giantswarm/cluster-apps-operator/pull/62 for example. 
![image](https://user-images.githubusercontent.com/2532687/129046973-95edc575-e487-493e-a9c5-e458f64e412b.png)

How can I successfully assign my team as reviewers to the PR? What possibly went wrong there?

**Comment by rarkins** [comment]
The log message you found comes from here: https://github.com/renovatebot/renovate/blob/9d580e091f1c3bd0084a10cd99c9df0ef9a78712/lib/workers/pr/index.ts#L102-L103

And in turn that function is here: https://github.com/renovatebot/renovate/blob/9d580e091f1c3bd0084a10cd99c9df0ef9a78712/lib/platform/github/index.ts#L1185-L1210

In theory that should have printed a WARN log if GitHub had rejected the request, so it's hard to know what went wrong.

**Reply by noahtallen** [reply]
I am also curious about this; I've had no luck getting it working:

https://github.com/Automattic/wp-calypso/blob/c78a382a296245be81aea14daafdb986e39874ab/renovate.json#L72

I think I tried the syntax mentioned here, but also `team:@Automattic/team-calypso-platform`

**Comment by noahtallen** [comment]
@tomahawk28 I've found a solution, finally! Here is the exact syntax: https://github.com/Automattic/wp-calypso/blob/651e07ac0bdec160247d9eeac3edf5f817a0bad8/renovate.json#L72

Specifically: `"reviewers": [ "team:team-calypso-platform" ]`. This team is now getting requested for new renovate PRs (though not for existing ones.)

I would ping this team in GitHub via `@Automattic/team-calypso-platform`, but renovate and/or GitHub won't accept that. It only wants the `team-calypso-platform` portion. This should really be documented by GitHub and/or Renovate

**Reply by rarkins** [reply]
You can contribute a PR to https://github.com/renovatebot/renovate/blob/main/docs/usage/configuration-options.md#reviewers

**Reply by HonkingGoose** [reply]
I've raised a PR, see: #12536. 😉

**Reply by JamieMagee** [reply]
And I merged it 🎉

**Reply by noahtallen** [reply]
> You can contribute a PR

Thanks for linking the docs, and thanks for quickly adding an example!
